pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -130,11 +129,8 @@ stdenv.mkDerivation rec {
     platforms = platforms.linux;
   };
 
-  passthru.updateScript = import ./../../browsers/firefox/update.nix {
-    name = "thunderbird";
-    sourceSectionRegex = ".";
-    basePath = "pkgs/applications/networking/mailreaders/thunderbird";
+  passthru.updateScript = callPackage ./../../browsers/firefox/update.nix {
```

Comment:
Should we have the same thing under a different name? It's quite painful to duplicate the list of upgrade script dependencies four times.